src/components/trials/steps/ParticipantsStep.tsx

```diff
@@ -44,13 +44,19 @@ const ParticipantsStep: React.FC = () => {
     const { trial, setHasChanged } = useTrialFormContext();
     const formInstance = useForm({ mode: "onChange" });
     const { getValues, handleSubmit } = formInstance;
-    const getParticipants = () => getValues({ nest: true });
+    const getParticipants = () => {
+        const participants = getValues({ nest: true })[KEY_NAME];
+        return {
+            [KEY_NAME]: participants.map((p: IParticipant) =>
+                !p.cohort_name ? omit(p, "cohort_name") : p
+            )
+        };
+    };
```

Comment:
This fixes a bug unrelated to this PR that was preventing participants with unspecified `cohort_name`s from being saved.